app/__init__.py

```diff
@@ -34,106 +40,197 @@
     }
 )
 
+# path navigating to current workspace directory
+WORKING_PATH = os.path.normpath(app.root_path + WORKING_DIR + r' ')
```

Comment:
@mansikulkarni96 Why do we need empty string appended ? at emd of WORKING_DIR ?

Reply:
We need a space because all workspace directories are created inside the app server in a parent directory which has no name, in order to navigate inside it a " " is needed.